workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/carousel-card/index.css

```diff
@@ -0,0 +1,122 @@
+.carousel-card {
+	background: var(--color-neutral-1, #F7F7F8);
+	border-radius: 0.75rem;
+	height: auto;  
+	padding: 2.5rem;
+	width: 100%;
+}
+
+.carousel-card-container {
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+}
+
+.carousel-card-information {
+	display: flex;
+	flex-direction: column;
+	gap: 1.5rem;
+	height: 100%;
+	justify-content: space-between;
+}
+
+.card-cta {
```

Comment:
Since `card-cta` is a generic class, it’s best to add it within a specific scope to avoid affecting other components.